aten/src/ATen/native/Normalization.cpp

```diff
@@ -54,6 +54,74 @@ struct Var {
   }
 };
 
+/// A fast path for CPU inference when all tensors are contiguous.
```

Comment:
I confirmed that having this file outside of `cpu/` emits SSE instructions, but surprisingly (to me) we're still hitting machine BW peak in my tests. For future proofing, maybe we should add a `TODO` to indicate we should enable newer vector extensions in the future, in case the balance of BW vs compute throughput on future CPU SKUs changes